_sass/_homepage.scss

```diff
@@ -165,4 +155,23 @@
             margin-bottom: 0;
         }
     }
+
+    @media (max-width: 600px) {
+        .key-points {
+            display: block;
+
+            p {
+                margin-bottom: 16px;
+            }
+        }
+ 
+        section {
+            padding: 12px;
```

Comment:
we strive for multiples of eight. When creating a new style, please use a multiple of eight.